src/index_runner/releng/samples.py

```diff
@@ -0,0 +1,136 @@
+'''
+Create a link for SampleSet
+'''
+import time
+import hashlib as _hashlib
+import datetime as _datetime
+# from collections import defaultdict as _defaultdict
+# import itertools as _itertools
+
+from src.utils.logger import logger
+from src.utils.re_client import stored_query as _stored_query
+from src.utils.re_client import save as _save
+from src.utils.re_client import get_doc as _get_doc
+from src.utils.re_client import clean_key as _clean_key
+from src.utils.sample_utils import get_sample as _get_sample
+from src.utils.re_client import MAX_ADB_INTEGER as _MAX_ADB_INTEGER
+
+
+# EXTREMELY HACKY FOR NOW. SHOULD BE RETRIEVED (?) FROM THE SAMPLE_SERVICE
+SAMPLE_COLL = 'samples_sample'
+SAMPLE_VER_COLL = 'samples_version'
+SAMPLE_NODE_COLL = 'samples_nodes'
+SAMPLE_ONTOLOGY_COLL = 'sample_ontology_link'
+
+# This should be information retrieved from the 'metadata_validation.yml'
+# file in the sample_service_validator_config
+SAMPLE_ONTOLOGY_VALIDATED_TERMS = [
+    ('biome', 'ENVO_terms'),
+    ('feature', 'ENVO_terms'),
+    ('ENIGMA:material', 'ENVO_terms')
+]
+
+
+def process_sample_set(obj_ver_key, obj_data):
+    """
+    :param obj_ver_key - object version key
```

Comment:
Let's use google style docstrings which look like this:
```
Args:
    obj_ver_key: object version key
    obj_data: object_data
```
And then type annotations can go in the actual code like this:
```py
def process_sample_set(obj_ver_key: str, obj_data: dict) -> None:
```